tests/snippets/bytearray.py

```diff
@@ -71,3 +71,17 @@
 a.append(100)
 assert len(a) == 6
 assert a.pop() == 100
+
+
+# repeat by multiply
+a = bytearray(b'abcd')
```

Comment:
Would adding `assert a * -1 == bytearray(b'')` be good for future?

Reply:
added